sdk/core/azure-core/CHANGELOG.md

```diff
@@ -1,9 +1,23 @@
 # Release History
 
-## 1.0.0-beta.2 (Unreleased)
+## 1.0.0-beta.2 (2020-10-08)
 
+### Breaking Changes
+
+- Throw Azure::Http::TransportException if creating new connection fails.
+- Response objects store Nullable<T>.
+
+### Bug Fixes
+
+- Switched to a more stable wait on sockets to address connection timeouts.
```

Comment:
Just double checking, does this correspond to https://github.com/Azure/azure-sdk-for-cpp/pull/676?

Reply:
Yes